docs/overview.md

```diff
@@ -25,25 +34,20 @@ As shown in the flow diagram above, each flow is comprised of a sequence of comp
 ![Component](images/parts_of_a_component.png)
 
 The component is made up of the following parts:
- - **Input Queue**: This is the queue that the component reads from. It is where the events are buffered as they flow through the system. Note that if there are multiple instances of the same component, they will all read from the same queue.
-
- - **Input Transforms**: This is an optional step that allows you to transform the event before it is processed. This can be useful for normalizing the data or for adding additional context to the event.
-
- - **Input Selection**: This selects what data should be processed by the component. The data selected should conform to the input schema of the component. It is normal to use Input Transforms and Input Selection together to ensure that the data is in the correct format for the component. If the Input Selection configuration is omitted, the component will select "previous" as the default, which will take the exact output of the previous component in the flow as the input to this component.
 
- - **Processing**: This is where the actual processing of the event happens, such as where the AI model would be called to process the event. This is the only required part of the component.
+- **Input Queue**: This is the queue that the component reads from. It is where the events are buffered as they flow through the system. Note that if there are multiple instances of the same component, they will all read from the same queue.
```

Comment:
We should say that it is a s/w queue inside the connector to make sure we don't give the impression that the queue is a Solace broker queue.

Reply:
done